components/authz-service/engine/opa/example_v2/real_world_2p1_store.json

```diff
@@ -1,361 +1,550 @@
-          "telemetry:*",
-          "iam:projects:list",
-          "iam:projects:get",
-          "iam:projects:assign",
-          "applications:*"
-        ]
+    "e729c61f-c40a-4bfa-affe-2a541368169f": {
+      "members": [
+        "tls:service:deployment-service:*"
+      ],
+      "statements": {
+        "0": {
+          "actions": [
+            "*"
+          ],
+          "effect": "allow",
+          "projects": [
+            "~~ALL-PROJECTS~~"
+          ],
+          "resources": [
+            "*"
+          ]
+        }
       },
-      "ingest": {
-        "actions": [
-          "infra:ingest:*",
-          "compliance:profiles:get",
-          "compliance:profiles:list"
-        ]
+      "type": "system"
+    },
+    "editor-access": {
+      "members": [
+        "team:local:editors"
+      ],
+      "statements": {
+        "0": {
+          "effect": "allow",
+          "projects": [
+            "~~ALL-PROJECTS~~"
+          ],
+          "resources": [
+            "*"
+          ],
+          "role": "editor"
+        }
       },
-      "owner": {
-        "actions": [
-          "*"
-        ]
+      "type": "chef-managed"
+    },
+    "ingest-access": {
+      "members": [],
+      "statements": {
+        "0": {
+          "effect": "allow",
+          "projects": [
+            "~~ALL-PROJECTS~~"
+          ],
+          "resources": [
+            "*"
+          ],
+          "role": "ingest"
+        }
       },
-      "project-owner": {
-        "actions": [
-          "infra:*",
-          "compliance:*",
-          "system:*",
-          "event:*",
-          "ingest:*",
-          "secrets:*",
-          "telemetry:*",
-          "iam:projects:list",
-          "iam:projects:get",
-          "iam:projects:assign",
-          "iam:policies:list",
-          "iam:policies:get",
-          "iam:policyMembers:*",
-          "iam:teams:list",
-          "iam:teams:get",
-          "iam:teamUsers:*",
-          "iam:users:get",
-          "iam:users:list"
-        ]
+      "type": "chef-managed"
+    },
+    "viewer-access": {
+      "members": [
+        "team:local:viewers"
+      ],
+      "statements": {
+        "0": {
+          "effect": "allow",
+          "projects": [
+            "~~ALL-PROJECTS~~"
+          ],
+          "resources": [
+            "*"
+          ],
+          "role": "viewer"
+        }
       },
-      "viewer": {
-        "actions": [
-          "secrets:*:get",
-          "secrets:*:list",
-          "infra:*:get",
-          "infra:*:list",
-          "compliance:*:get",
-          "compliance:*:list",
-          "system:*:get",
-          "system:*:list",
-          "event:*:get",
-          "event:*:list",
-          "ingest:*:get",
-          "ingest:*:list",
-          "iam:projects:list",
-          "iam:projects:get",
-          "applications:*:list",
-          "applications:*:get"
-        ]
-      }
+      "type": "chef-managed"
+    },
+    "events-access-legacy": {
```

Comment:
decided to add migrated legacy policies to the real world policy examples json. This json file now includes all baseline policies a typical V2 Automate might have: system policies (hidden from customers), Chef-managed policies, and migrated V1 default policies.